dom-event-delegation/main.js

```diff
@@ -0,0 +1,9 @@
+
+var taskList = document.querySelector('.task-list');
+
+function handleClick(event) {
+  if (event.target.matches('button')) {
```

Comment:
This works, just don't forget the console logs!

Reply:
I had removed the console logs before committing because I remembered a different exersize where the linter threw it back at me. I'll re add them again.